release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2622,3 +2622,12 @@ link:https://access.redhat.com/solutions/6308491[{product-title} 4.8.10 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-8-preparing-upgrade-next-release"]
+==== Preparing to upgrade to the next {product-title} release
+
+{product-title} 4.8.10 introduced a check that will impact all upgrades to the next {product-title} release, currently planned as {product-title} 4.9. This is because Kubernetes 1.22 removed a significant number of deprecated `v1beta1` APIs. This check is to try to prevent issues after upgrading where the removed APIs are still in use.
```

Comment:
so we don't forget: 4.8.10 -> whenever [rhbz#1999092](https://bugzilla.redhat.com/show_bug.cgi?id=1999092) lands in 4.8.z

Reply:
Right right, this was just a test of how it might look. Thanks for mentioning it just in case!